src/scripts/settings.ts

```diff
@@ -3,13 +3,13 @@ import locales from "./i18n/_locales"
 import { ThemeSettings } from "../schema-types"
 import intl from "react-intl-universal"
 
-const lightTheme: IPartialTheme = { 
-    defaultFontStyle: { fontFamily: '"Segoe UI", "Source Han Sans SC Regular", "Microsoft YaHei", sans-serif' } 
+const lightTheme: IPartialTheme = {
+    defaultFontStyle: { fontFamily: '"Segoe UI", "Source Han Sans SC Regular", "Microsoft YaHei", sans-serif' }
 }
 const darkTheme: IPartialTheme = {
     ...lightTheme,
     palette: {
-        neutralLighterAlt: "#282828",
+        neutralLighterAlt: "#282828CC",
```

Comment:
Please do not change the global palette.

Reply:
Reverted https://github.com/yang991178/fluent-reader/pull/72/commits/b2a19eb6156f55eacc41590341226fb087fcba8e .